File: ios/Plugin/GooglePlayAvailabilityPlugin.m
#import <Foundation/Foundation.h>
#import <Capacitor/Capacitor.h>

// Define the plugin using the CAP_PLUGIN Macro, and
// each method the plugin supports using the CAP_PLUGIN_METHOD macro.
CAP_PLUGIN(GooglePlayAvailabilityPlugin, "GooglePlayAvailability",
           CAP_PLUGIN_METHOD(isGooglePlayServicesAvailable, CAPPluginReturnPromise);
           CAP_PLUGIN_METHOD(makeGooglePlayServicesAvailable, CAPPluginReturnPromise);
)

File: ios/Plugin/GooglePlayAvailabilityPlugin.swift
import Foundation
import Capacitor

/**
 * Please read the Capacitor iOS Plugin Development Guide
 * here: https://capacitorjs.com/docs/plugins/ios
 */
@objc(GooglePlayAvailabilityPlugin)
public class GooglePlayAvailabilityPlugin: CAPPlugin {

    @objc func isGooglePlayServicesAvailable(_ call: CAPPluginCall) {
        call.unimplemented()
    }

    @objc func makeGooglePlayServicesAvailable(_ call: CAPPluginCall) {
        call.unimplemented()
    }
}
